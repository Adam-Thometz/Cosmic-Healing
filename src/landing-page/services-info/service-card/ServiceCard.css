.ServiceCard {
  background-color: white;
  border-radius: 10px;
  text-align: left;
  padding: 2% 3%;
  height: 100%;
  width: 25%;
  margin: 10px auto;
  font-size: 18px;
  font-family: Nuckle;
  font-weight: 550;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  box-shadow: 0px 4px 35px 0px #782DCA26;
}

.ServiceCard .Link {
  width: 85%;
  display: flex;
  justify-content: center;
}