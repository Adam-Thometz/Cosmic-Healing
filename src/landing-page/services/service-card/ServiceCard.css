.ServiceCard {
  background-color: white;
  border-radius: 10px;
  text-align: left;
  padding: 2% 3%;
  height: 100%;
  width: 20%;
  margin: 0 auto;
  font-size: 18px;
  font-family: Nuckle;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
}

.ServiceCard button {
  width: 66%;
}