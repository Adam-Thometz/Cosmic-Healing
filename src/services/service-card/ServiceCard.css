.ServiceCard {
  background-color: white;
  border-radius: 10px;
  text-align: left;
  padding: 24px 32px;
  /* height: 300px; */
  width: 20%;
  margin: 0 auto;
}