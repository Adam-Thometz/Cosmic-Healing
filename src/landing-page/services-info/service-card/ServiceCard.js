import React from 'react'

import './ServiceCard.css';

import Link from '../../../_components/link/Link';
import SectionHeader from '../../../_components/section-header/SectionHeader';

const ServiceCard = ({ service, description, link }) => {
  return (
    <div className='ServiceCard'>
      <SectionHeader small text={service} />
      <article className='ServiceCard-description'>{description}</article>
      <Link secondary to={link} >Learn more</Link>
    </div>
  );
};

export default ServiceCard;