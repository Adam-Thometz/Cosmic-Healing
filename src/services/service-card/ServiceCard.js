import React from 'react'

import './ServiceCard.css';

import Button from '../../_components/button/Button'
import SectionHeader from '../../_components/section-header/SectionHeader'

const ServiceCard = ({ service, description, link }) => {
  return (
    <div className='ServiceCard'>
      <SectionHeader small text={service} />
      <article className='ServiceCard-description'>{description}</article>
      <Button>Learn more</Button>
    </div>
  )
}

export default ServiceCard