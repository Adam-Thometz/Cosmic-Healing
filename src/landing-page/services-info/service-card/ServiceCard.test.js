import React from "react";
import { render } from "@testing-library/react";
import ServiceCard from './ServiceCard';

import options from '../serviceOptions'

describe('ServiceCard component', () => {
  const { service, description, link } = options[0]
  const component = <ServiceCard
    service={service}
    description={description}
    link={link}
  />;
  it('renders without crashing', () => {
    render(component);
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(component);
    expect(asFragment()).toMatchSnapshot();
  });
});