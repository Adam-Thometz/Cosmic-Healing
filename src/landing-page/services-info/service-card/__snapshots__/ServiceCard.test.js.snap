// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServiceCard component matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="ServiceCard"
  >
    <h3
      class="SectionHeader"
    >
      Individual Quest
    </h3>
    <article
      class="ServiceCard-description"
    >
      Solo journey.
    </article>
    <button
      class="Button secondary"
    >
      Learn more
      <img
        alt=""
        class="Button-arrow"
        src="arrow.png"
      />
    </button>
  </div>
</DocumentFragment>
`;
